source/guide_cloudlog.rst

```diff
@@ -0,0 +1,138 @@
+
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PHP_
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.4:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.4'
+ [isabell@stardust ~]$
+
+Your log domain needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+First, remove the default ``nocontent.html`` from your :manual:`document root <web-documentroot>` and delete the now empty ``html`` folder:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ rm /var/www/virtual/$USER/html/nocontent.html
+  [isabell@stardust ~]$ rmdir /var/www/virtual/$USER/html
+  [isabell@stardust ~]$
+
+Now use ``git`` to clone the Cloudlog repo into a new ``html`` folder:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ git clone https://github.com/magicbug/Cloudlog.git /var/www/virtual/$USER/html/
+  Cloning into '/var/www/virtual/$USER/html'...
+  remote: Enumerating objects: 24402, done.
+  remote: Counting objects: 100% (370/370), done.
+  remote: Compressing objects: 100% (149/149), done.
+  remote: Total 24402 (delta 247), reused 320 (delta 219), pack-reused 24032
+  Receiving objects: 100% (24402/24402), 16.23 MiB | 23.31 MiB/s, done.
+  Resolving deltas: 100% (15879/15879), done.
+  [isabell@stardust ~]$
+
+Set up a MySQL database for your Cloudlog installation:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ mysql -e "CREATE DATABASE ${USER}_cloudlog;"
+  [isabell@stardust ~]$
+
+Configuration
+=============
+
+Point your browser to Cloudlog's setup wizard at ``https://<yourusername>.uber.space/install`` (in our example, this would be ``https://isabell.uber.space/install``).
+
+.. include:: includes/my-print-defaults.rst
+
+In the web form, you can leave the ``Directory`` field blank if you plan on hosting your Cloudlog instance at the root folder (i. e. ``https://isabell.uber.space/``) of your domain. The ``Website URL`` should already be pre-filled with your site's URL. You can change the ``Default Gridsquare`` if you like.
+
+For the ``Database settings``, use the hostname ``localhost``, your Uberspace user name as ``Username`` and your MySQL password as ``password``. Enter the ``Database name`` you created in the previous step (in our example this would be ``isabell_cloudlog``). Once all the required information has been entered, click install. You should now be redirected to your Cloudlog's home page. Log in with the default username ``m0abc`` and default password ``demo``, then open Admin -> User Accounts and create a new user. In order to delete the default user, log out and then log in again with your newly created Cloudlog user, head to Admin -> User Accounts again and delete ``m0abc``.
```

Comment:
did you check if its possible to do all this via terminal and in config files? Its a bit hard to follow. If its neccessary to do in web maybe a numbered list would be more clearly

Reply:
I'm aware of it being a bit confusing, but couldn't find any CLI setup method for this in the [[.](https://github.com/magicbug/Cloudlog/wiki/Installation#5-run-the-cloudlog-install-wizard)](official documentation). A numbered list sounds like a good idea though.